src/components/tutor-popover.cjsx

```diff
@@ -0,0 +1,116 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+ArbitraryHtml = require './html'
+
+TutorPopover = React.createClass
```

Comment:
not quick drop-in but i think we could move in that direction after this gets merged